package/docker/Dockerfile

```diff
@@ -46,3 +46,5 @@ RUN    sudo apt-get update        \
 
 ADD --chown=user:user ./kevm-pyk /home/user/kevm-pyk
 RUN cd /home/user/kevm-pyk && pip3 install .
+
+ENV PATH=${PATH}:/home/user/.local/bin
```

Comment:
Is this something that we should have done before, or there's a new file that is put there?

Reply:
Instead of doing `python3 -m kevm-pyk ...`, we are just doing directly `kevm-pyk ...`. So the direct call to the binary means that we need to have it on PATH.